keras/random/seed_generator_test.py

```diff
@@ -77,3 +77,31 @@ def traced_function():
             "When tracing a JAX function, you should only use seeded random",
         ):
             traced_function()
+
+    def test_seed_generator_serialization(self):
+        seed = 42
+        random_generator = seed_generator.SeedGenerator(seed=seed)
+        config = random_generator.get_config()
+        self.assertEqual(config["seed"], seed)
+
+        reconstructed_random_generator = random_generator.from_config(config)
+
+        def config_equals(config1, config2):
```

Comment:
You can just use `self.run_class_serialization_test(random_generator)` and you'll be done

Reply:
:') Awesome! updated the tests!